java-test-samples/apigw-lambda-list-s3-buckets/src/test/java/com/example/AppTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT-0
+ */
+
+package com.example;
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(SystemStubsExtension.class)
+public class AppTest {
+  @SystemStub
+  private final EnvironmentVariables environmentVariables = new EnvironmentVariables();
+
+  @BeforeEach
+  public void setup() {
+    environmentVariables.set("AWS_REGION", "us-east-2");
+  }
+
+  @Test
+  public void successfulResponse() {
+    App app = new App();
+    APIGatewayProxyResponseEvent result = app.handleRequest(null, null);
```

Comment:
Not realistic (use events and lambda-test library)

Reply:
Fixed